Editor/CoreAPI/GameLiftFleetManager.cs

```diff
@@ -152,7 +151,7 @@ public async Task<List<FleetAttributes>> ListFleetAttributes()
                     FleetIds = listFleetResponse.FleetIds
                 };
 
-                var describeFleetResponse = await _amazonGameLiftWrapper.DescribeFleetAttributes(describeFleetRequest);
+                var describeFleetResponse = await _amazonGameLiftWrapper.DescribeFleets(describeFleetRequest);
```

Comment:
Shouldn't this be `DescribeFleetAttributes`? I don't see a `DescribeFleets` method in the wrapper. Also, we shouldn't introduce `DescribeFleets` and prefer `DescribeFleetAttributes` to match the API name

Reply:
This method has been changed in another PR, don't want to change anything until there is a merge. Keeping comment Open

Reply:
https://github.com/aws/amazon-gamelift-plugin-unity/pull/111#discussion_r1345753038